src/components/SettingsPage/Settings.tsx

```diff
@@ -37,8 +38,12 @@ const StyledTextFieldContainer = styled.div`
 
 function Settings() {
     const [isOpen, setIsOpen] = React.useState(false);
+    const settingsContext = useContext(SettingsContext);
 
-    function saveSettings() {
+    function saveSettings(e: React.FormEvent<HTMLFormElement>) {
+        e.preventDefault();
+        settingsContext.setDOB(new Date((document.getElementsByName('dob')[0] as HTMLInputElement).value));
+        settingsContext.setLifeExpectancy((document.getElementsByName('life-expectancy')[0] as HTMLInputElement).valueAsNumber);
```

Comment:
I'm thinking it might be possible to get these values from the `e`

Reply:
Addressed